corehq/ex-submodules/couchforms/analytics.py

```diff
@@ -277,17 +280,24 @@ def get_exports_by_form(domain):
 
 
 def get_form_count_breakdown_for_domain(domain):
-    rows = XFormInstance.get_db().view(
-        'exports_forms_by_xform/view',
-        startkey=[domain],
-        endkey=[domain, {}],
-        group=True,
-    )
-    result = {}
-    for row in rows:
-        domain, app_id, xmlns = row['key']
-        result[(domain, app_id, xmlns)] = row['value']
-    return result
+    query = (
+        FormES()
+            .domain(domain)
+            .aggregation(
+                TermsAggregation("app_id", "app_id").aggregation(
+                    TermsAggregation("xmlns", "xmlns.exact")))
+            .remove_default_filter("has_xmlns")
+            .remove_default_filter("has_user")
+            .size(0))
+    query_result = query.run()
+    buckets = query_result.aggregations.app_id.normalized_buckets
+    form_counts = {}
+    for bucket in buckets:
+        app_id = bucket['key']
+        for sub_bucket in bucket['xmlns']['buckets']:
```

Comment:
Ah yeah, so I'm using `BucketResult.normalized_buckets`, which is just a dict, because there is some issue wrapping the result when using double aggregation that I haven't figured out yet. But I should, so I'll take a look into that...